app/src/androidTest/java/org/mozilla/focus/helpers/StringsHelper.kt

```diff
@@ -13,3 +13,8 @@ object StringsHelper {
     const val EN_LANGUAGE_MENU_HEADING = "Language"
     const val EN_FRENCH_LOCALE = "Français"
 }
+
+object PackageName {
+    const val GMAIL_APP = "com.google.android.gm"
+    const val PHONE_APP = "com.android.dialer"
```

Comment:
could we add these to the list above and drop the PackageName object?

Reply:
Sure, done.